samcli/commands/list/exceptions.py

```diff
@@ -6,7 +6,11 @@
 from samcli.commands.exceptions import UserException
 
 
-class StackOutputsError(UserException):
+class SamListError(UserException):
+    """
+    Base exception for the 'sam list' command
+    """
+
```

Comment:
nit: I think we don't need this `__init__` method here? Can you remove and verify if we need it or not?

Reply:
I removed it and got errors because there's I can't pass in the msg anymore